packages/marlinreco/package.py

```diff
@@ -37,6 +37,8 @@ class Marlinreco(CMakePackage, Ilcsoftpackage):
         sha256="0ea3bee03e2bec1924b5876675043b592a942bc8cf306eb7056eaf03ac1748f6",
     )
 
+    patch("algorithm.patch", when="@1.36")
```

Comment:
```suggestion
    patch("algorithm.patch", when="@:1.36")
```
Given that the file hasn't been changed in several years, let's apply it to every version that we have. By the way, I see now that 1.36 isn't even in the recipe.